### PR TITLE
feat(ui): Add minicart empty state

### PR DESCRIPTION
**Pull request by saranicoly**
## What's the purpose of this pull request?
The Jira task can be seen [here](https://vtex-dev.atlassian.net/jira/software/projects/FSSS/boards/632?selectedIssue=FSSS-136).

This is how it's looking right now: image

## How it works? 
When the cart is opened and there is no selected item (which means the cart is empty), a message should be shown. Clicking in 'start shopping', the cart is closed.

## How to test it?
This is not being used yet, so if you want to see how it's looking, put the code in `EmptyCart.tsx` inside another file, such as `CartSideBar.tsx`. 

_Thanks @renatamottam._

### CODE REVIEW

**Review by eduardoformiga — DISMISSED** [review]
Good Job! 👏 
Let's wait for the styled version then.

File: src/components/cart/EmptyCart/EmptyCart.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import Button from 'src/components/ui/Button'
+import { ShoppingCart as ShoppingCartIcon } from 'phosphor-react'
+import { useCartToggleButton } from 'src/sdk/cart/useCartToggleButton'
+
```
Comment (eduardoformiga) [inline src/components/cart/EmptyCart/EmptyCart.tsx:5]:
```suggestion

import './empty-cart.scss '

```

**Review by eduardoformiga — CHANGES_REQUESTED** [review]
Good Job 🎉 .
I miss the style file into the component.

**Review by filipewl — APPROVED** [review]

File: src/components/cart/EmptyCart/EmptyCart.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+function EmptyCart() {
+  const toggleProps = useCartToggleButton()
+
+  return (
+    <div data-testid="cart-empty-state" data-empty-cart>
+      <div data-empty-cart-title>
+        <ShoppingCartIcon size="32" />
+        <p>Your Cart is empty</p>
+      </div>
+      <Button {...toggleProps} variant="primary">
```
Comment (filipewl) [inline src/components/cart/EmptyCart/EmptyCart.tsx:15]: I haven't checked, but what do those props do?

File: src/components/cart/EmptyCart/EmptyCart.tsx (hunk 1 above)
Comment (saranicoly) [reply src/components/cart/EmptyCart/EmptyCart.tsx:5]: Oh, thanks for seeing that! 
I totally forgot about it.

File: src/components/cart/EmptyCart/EmptyCart.tsx (hunk 2 above)
Comment (saranicoly) [reply src/components/cart/EmptyCart/EmptyCart.tsx:17]: It's to close the cart. It does the same thing as the `close` button in `CartSidebar.tsx`.

**Review by eduardoformiga — APPROVED** [review]

**Review by danzanzini — APPROVED** [review]
🎉

**Review by israelswf — APPROVED** [review]
✨

**Pull request merged**